runhouse/resources/hardware/cluster.py

```diff
@@ -645,9 +645,10 @@ def _sync_runhouse_to_cluster(
             # rh_package = 'runhouse_nightly-0.0.1.dev20221202-py3-none-any.whl'
             # rh_download_cmd = f'curl https://runhouse-package.s3.amazonaws.com/{rh_package} --output {rh_package}'
             if not _install_url:
-                import runhouse
+                # import runhouse
 
-                _install_url = f"runhouse=={runhouse.__version__}"
+                # _install_url = f"runhouse=={runhouse.__version__}"
+                _install_url = "git+https://github.com/run-house/runhouse.git@10-25-updates_cluster_teardown_via_den#egg=runhouse[all]"
```

Comment:
just need to revert this, otherwise looks the PR lgtm!

Reply:
Have to leave this in here temporarily for testing, so that when the launcher deploys this branch of runhouse, it's able to deploy it to the launched clusters as well